modules/tinymce/src/core/main/ts/DragDropOverrides.ts

```diff
@@ -121,6 +132,75 @@ const moveGhost = (
 
   ghostElm.style.width = (width - overflowX) + 'px';
   ghostElm.style.height = (height - overflowY) + 'px';
+
+  // Code needed for dragging CEF elements (specifically fixing TINY-8874)
+  // The idea behind the algorithm is that the user will start dragging the
+  // CEF element to the edge of the editor and that would cause scrolling.
+  // The way that happens is that the user will trigger a mousedown event,
+  // then a mousemove event until they reach the edge of the editor. Then
+  // no event triggers. That's when I set an interval to keep scrolling the editor.
+  // Once a new event triggers I clear the existing interval and set it back to none.
+
+  const clientHeight = contentAreaContainer.clientHeight;
+  const clientWidth = contentAreaContainer.clientWidth;
+
+  state.on((state) => {
+    if (state.dragging) {
+      // This basically means that the mouse is close to the bottom edge
+      // (within MouseRange pixels of the bottom edge)
+      if (mouseY + MouseRangeToTriggerScroll >= clientHeight) {
+        const scrollDown = (currentTop: number) => {
+          win.scroll({
```

Comment:
Have you tested this works with say the autoresize plugin enabled? As the window is always the same height so it's rather unlikely using window.scroll() will work in that case.